api/routers/areas.py

```diff
@@ -36,6 +36,13 @@ async def get_area(area_id: str):
     """
     Returns the configuration related to the area <area_id>
     """
+    if area_id.upper() == "ALL":
```

Comment:
Maybe we can create a constant called ALL_AREAS with the value "ALL"  and use It instead of repeat the string,

Reply:
Good idea!